app/src/test/java/nl/tudelft/b_b_w/controller/BlockControllerUnitTest.java

```diff
@@ -242,6 +263,69 @@ public void testRevokedTrustValue() {
         assertEquals(TrustValues.REVOKED.getValue(), _block.getTrustValue());
     }
 
+    /** Check that the genesis block is created */
+    @Test
+    public void verifyGenesisCreation() {
+        assertNotNull(genesisA);
+    }
+
+    /** Check that genesis has the correct index */
+    @Test
+    public void verifyGenesisIndex() {
+        assertEquals(1, genesisA.getSequenceNumber());
+    }
+
+    /** Verify genesis owner A */
+    @Test
+    public void verifyGenesisOwner() {
+        assertEquals("A", genesisA.getOwner());
+    }
+
+    /** Hash of genesis block */
+    @Test
+    public void verifyGenesisHashA() {
+        ConversionController conversionController = new ConversionController("A", "N/A", "N/A",
+                "N/A", "N/A");
+        try {
+            String hash = conversionController.hashKey();
+            assertEquals(genesisA.getOwnHash(), hash);
+        } catch (Exception e) {
+            assertNotNull(e.getMessage(), null);
+        }
+    }
+
+    /** Verify the block A-A1 was created */
+    @Test
+    public void verifyBlockA_A1Creation() {
```

Comment:
What is A_A1? Can you change this and be more specifiek. The attribute names are pretty important.